### PR TITLE
Add support for simple table

### PR DESCRIPTION
**Pull request by enpitsuLin**
inspired by #166 

add block type and also add some simple stylesheet

simple-table

***
also noticed default css variable is not variety enough just like `fg-color-1(~5)`

### CODE REVIEW

**Comment by normdoow** [comment]
Hey @enpitsuLin, thanks for adding this. Awesome! I will take a look in the next couple of days.

**Comment by imsukmin** [comment]
When did you deploy it? I need this function.

**Comment by normdoow** [comment]
> When did you deploy it? I need this function.

This hasn't been deployed yet. But Hopefully tomorrow if everything is good.

File: packages/react-notion-x/src/block.tsx [hunk 1]
```diff
@@ -832,6 +832,37 @@ export const Block: React.FC<BlockProps> = (props) => {
         </components.pageLink>
       )
 
+    case 'table':
+      return (
+        <table className='notion-simple-table'>
```
Comment (normdoow) [inline packages/react-notion-x/src/block.tsx:837]: Can you add the notion blockId as a class? Like `className={cs(blockId)}`. You'll see most of the other blocks also have this. This is so people can target specific blocks and customize it with css.

File: packages/react-notion-x/src/block.tsx [hunk 2]
```diff
@@ -832,6 +832,37 @@ export const Block: React.FC<BlockProps> = (props) => {
...
+        </table>
+      )
+    case 'table_row':
+      const tableBlock = recordMap.block[block.parent_id]
+        .value as types.TableBlock
+      const order = tableBlock.format.table_block_column_order
+      const formatMap = tableBlock.format.table_block_column_format
+
+      return (
+        <tr className='notion-simple-table-row'>
```
Comment (normdoow) [inline packages/react-notion-x/src/block.tsx:848]: Can you add the notion block id to the rows as well? `className={cs(blockId)}`.

File: packages/react-notion-x/src/block.tsx (hunk 1 above)
Comment (enpitsuLin) [reply packages/react-notion-x/src/block.tsx]: got it.just finished

**Comment by normdoow** [comment]
Live for v4.14.0! Thanks @enpitsuLin.
One thing I did see was missing that would be nice to add is styling based on the table row header and column header setting in Notion.

**Pull request merged**